back-end/java/main/java/src/DormRoom/BathroomType.java

```diff
@@ -0,0 +1,20 @@
+package src.DormRoom;
+
+/**
+ * Enum of possible bathroom types for a dorm room.
+ */
+public enum BathroomType {
```

Comment:
One note. I believe there are some rooms with only sink, only sink and toilet, and then only sink toilet and shower. These are all different types of "private" bathrooms" that we will see later when the dataset is complete. We can decide how we use that data. Just something to keep in mind when further using this enum to make it easy to change if we add more states.